### ISSUE TITLE
icc_profile

### ISSUE THREAD

**Issue by radarhere** [python-pillow/Pillow#1778]
from PIL import Image
    Image.open('Pillow/Tests/images/hopper.iccprofile.tif').save('test.png')

In Python 3.5, this results in

    Traceback (most recent call last):
      Line 2, in <module>
        Image.open('Pillow/Tests/images/hopper.iccprofile.tif').save('test.png')
      File "PIL/Image.py", line 1675, in save
        save_handler(self, fp, filename)
      File "PIL/PngImagePlugin.py", line 757, in _save
        data = name + b"\0\0" + zlib.compress(im.info["icc_profile"])
    TypeError: a bytes-like object is required, not 'tuple'

In Python 3.3 and 3.4, it's `TypeError: 'tuple' does not support the buffer interface`. In Python 2.7, it's `TypeError: must be string or read-only buffer, not tuple`

`im.info["icc_profile"]` is a tuple of length 1, where the first argument is a bytes-like object.

**Comment by wiredfool** [comment]
Is this a recent break, or is it something that's been around a while?  We did touch some profile stuff lately, but I don't think it's part of this code path.

**Comment by radarhere** [comment]
Good question. For whatever reason, I was only thinking about what the best solution was, rather than wondering if it had always been a bug.

Testing, I find that the problem occurred only after PR #1419 was merged.

**Comment by wiredfool** [comment]
Ah, the unintentional break everything commit.

**Comment by radarhere** [comment]
I've realised this is a duplicate of the closed issue #1462

**Comment by hugovk** [comment]
I've reopened #1462 because it's reproducible with latest Pillow 3.2.0: https://github.com/python-pillow/Pillow/issues/1462#issuecomment-204833624

But let's close this duplicate.
